packages/dashboard/src/customization/widgets/types.ts

```diff
@@ -90,6 +90,13 @@ export type TextProperties = {
   href?: string;
 };
 
+export type RectangleProperties = {
+  style: 'solid' | 'dashed' | 'dotted';
+  fill: string;
+  strokeColor: string;
```

Comment:
these names are kind of specific to svg properties. What about something like color / thickness / backgroundColor? they would be more similar to the property definition we established for the new chart component

Reply:
are these all required props or can we make some optional and set defaults?

Reply:
> these names are kind of specific to svg properties. What about something like color / thickness / backgroundColor? they would be more similar to the property definition we established for the new chart component

right. i think we can keep `fill` as is. we can't make it `fillColor` since 'none' is an acceptable value for it. however, i do agree we can change the others:

`style` --> `borderStyle`
`strokeColor` --> `borderColor`
`strokeWidth` --> `borderThickness`

Reply:
> are these all required props or can we make some optional and set defaults?

good point. we can make all of them optional.